### PR TITLE
Chore/next poc code cleanup

### PR DESCRIPTION
**Pull request by GurdeepSSingh**
- [] Before opening a pull request, please read ApeSwap's [contributing guidelines](https://github.com/ApeSwapFinance/apeswap-frontend/blob/staging/CONTRIBUTING.md).
- [] Explain what your PR does & please link it to a grant/issue if applicable.
- [] If your PR is for a grant, please provide contact & fund recipient information in the PR.
- [] If your PR is work in progress, open it as a `draft`.

### CODE REVIEW

File: src/components/LiquidityWidget/ConnectButton.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const ConnectButton = () => {
   const { onPresentConnectModal } = useWalletModal(login, logout, t)
 
   return (
-    <Button onClick={onPresentConnectModal} csx={styles.button}>
+    <Button onClick={onPresentConnectModal} sx={styles.button}>
```
Comment (kil-san) [inline src/components/LiquidityWidget/ConnectButton.tsx:15]: I don't see the pragma at the top of this file

File: src/components/LiquidityWidget/LiquidityWidget.tsx [hunk 2]
```diff
@@ -486,7 +445,7 @@ const LiquidiyWidget: React.FC<ILiquidityWidgetProps> = ({ onCancel }) => {
                   <Flex sx={{ justifyContent: 'space-between', columnGap: '15px' }}>
                     {approvalA !== ApprovalState.APPROVED && (
                       <Button
-                        csx={styles.button}
+                        sx={styles.button}
```
Comment (kil-san) [inline src/components/LiquidityWidget/LiquidityWidget.tsx:449]: no pragma in this file too, can you check others too

**Review by kil-san — CHANGES_REQUESTED** [review]

File: src/components/Menu/components/ConnectButton.tsx [hunk 3]
```diff
@@ -18,6 +18,7 @@ const ConnectButton: React.FC<any> = () => {
   const { isDark } = useTheme()
 
   const buttonStyle = dynamicStyles.userBlockBtn({ account })
+  const iconstyleconnect = dynamicStyles.connectButtonIcon({})
```
Comment (kil-san) [inline src/components/Menu/components/ConnectButton.tsx]: why is this dynamic?

File: src/components/Menu/components/DesktopMenu.tsx [hunk 4]
```diff
@@ -93,12 +91,7 @@ const DesktopMenu: React.FC<DesktopMenuProps> = ({ items }) => {
                   <ImageElement />
                   <Flex
                     sx={{
-                      justifyContent: 'center',
-                      alignContent: 'center',
-                      columnGap: '10px',
-                      background: 'navbar',
-                      borderRadius: '25px',
-                      height: '50px',
+                      ...socialMedia,
```
Comment (kil-san) [inline src/components/Menu/components/DesktopMenu.tsx]: we don't need to spread this

File: src/components/Menu/index.tsx [hunk 5]
```diff
@@ -107,13 +105,8 @@ const Menu: React.FC<{ chain?: number }> = () => {
...
-          position: 'fixed',
-          width: '100%',
-          height: '60px',
+          ...styles.header,
           transform: `translateY(${showMenu ? 0 : '-100%'})`,
-          transition: 'transform linear 0.1s',
-          zIndex: 9,
         }}
       >
         <Flex
```
Comment (kil-san) [inline src/components/Menu/index.tsx]: sx props here is not cleaned up

File: src/components/NetworkButton/NetworkButton.tsx [hunk 6]
```diff
@@ -40,11 +41,7 @@ const NetworkButton: React.FC = () => {
       variant="tertiary"
       background="white3"
       sx={{
-        display: 'flex',
-        alignItems: 'center',
-        px: '11px',
-        py: '8px',
-        columnGap: '8px',
+        ...styles.button,
```
Comment (kil-san) [inline src/components/NetworkButton/NetworkButton.tsx]: no need to spread here

File: src/views/Bills/components/Menu/index.tsx [hunk 7]
```diff
@@ -50,7 +50,7 @@ const BillMenu: React.FC<ListViewProps> = ({ onHandleQueryChange, onSetSortOptio
       </Flex>
       <Link href="?modal=bills" shallow>
         <Button
-          csx={{ height: '36px', width: '192px', border: '2px solid', justifyContent: 'center' }}
+          sx={{ height: '36px', width: '192px', border: '2px solid', justifyContent: 'center' }}
```
Comment (kil-san) [inline src/views/Bills/components/Menu/index.tsx]: style should be moved out of component

File: src/views/DualFarms/components/DisplayFarms.tsx [hunk 8]
```diff
@@ -1,3 +1,4 @@
+/** @jsxImportSource theme-ui */
```
Comment (kil-san) [inline src/views/DualFarms/components/DisplayFarms.tsx]: remove from here

File: src/views/Farms/components/DisplayFarms.tsx [hunk 9]
```diff
@@ -1,3 +1,4 @@
+/** @jsxImportSource theme-ui */
```
Comment (kil-san) [inline src/views/Farms/components/DisplayFarms.tsx]: remove

File: src/views/Nft/components/NftCard/index.tsx [hunk 10]
```diff
@@ -1,3 +1,4 @@
+/** @jsxImportSource theme-ui */
```
Comment (kil-san) [inline src/views/Nft/components/NftCard/index.tsx]: remove

File: src/pages/_app.tsx [hunk 11]
```diff
@@ -1,3 +1,4 @@
+/** @jsxImportSource theme-ui */
```
Comment (kil-san) [inline src/pages/_app.tsx]: remove

File: src/components/layout/HeaderSection.tsx [hunk 12]
```diff
@@ -1,3 +1,4 @@
+/** @jsxImportSource theme-ui */
```
Comment (kil-san) [inline src/components/layout/HeaderSection.tsx]: remove

File: src/components/Menu/components/MobileMenu.tsx [hunk 13]
```diff
@@ -38,11 +39,7 @@ const MobileMenu: React.FC<MobileMenuProps> = ({ items }) => {
           transition={{ height: { duration: 0.3 } }}
           exit={{ height: 0 }}
           sx={{
-            width: '100%',
-            background: 'navbar',
-            position: 'absolute',
-            overflow: 'hidden',
-            zIndex: 10,
+            menuMobile,
```
Comment (kil-san) [inline src/components/Menu/components/MobileMenu.tsx]: this should be `sx={menuMobile}`

**Review by kil-san — CHANGES_REQUESTED** [review]

File: src/components/Menu/components/DesktopMenu.tsx (hunk 4 above)
Comment (GurdeepSSingh) [reply src/components/Menu/components/DesktopMenu.tsx]: Done

File: src/views/DualFarms/components/DisplayFarms.tsx (hunk 8 above)
Comment (GurdeepSSingh) [reply src/views/DualFarms/components/DisplayFarms.tsx]: Done

File: src/views/Farms/components/DisplayFarms.tsx (hunk 9 above)
Comment (GurdeepSSingh) [reply src/views/Farms/components/DisplayFarms.tsx]: Done

File: src/views/Nft/components/NftCard/index.tsx (hunk 10 above)
Comment (GurdeepSSingh) [reply src/views/Nft/components/NftCard/index.tsx]: Done

File: src/components/layout/HeaderSection.tsx (hunk 12 above)
Comment (GurdeepSSingh) [reply src/components/layout/HeaderSection.tsx]: Done

File: src/pages/_app.tsx (hunk 11 above)
Comment (GurdeepSSingh) [reply src/pages/_app.tsx]: Done

File: src/components/Menu/components/MobileMenu.tsx (hunk 13 above)
Comment (GurdeepSSingh) [reply src/components/Menu/components/MobileMenu.tsx]: Done

File: src/views/Bills/components/Menu/index.tsx (hunk 7 above)
Comment (GurdeepSSingh) [reply src/views/Bills/components/Menu/index.tsx]: Done

File: src/components/Menu/components/ConnectButton.tsx (hunk 3 above)
Comment (GurdeepSSingh) [reply src/components/Menu/components/ConnectButton.tsx]: Removed Dynamic

File: src/components/LiquidityWidget/ConnectButton.tsx (hunk 1 above)
Comment (GurdeepSSingh) [reply src/components/LiquidityWidget/ConnectButton.tsx:15]: Done

File: src/components/LiquidityWidget/LiquidityWidget.tsx (hunk 2 above)
Comment (GurdeepSSingh) [reply src/components/LiquidityWidget/LiquidityWidget.tsx:449]: Done

File: src/components/NetworkButton/NetworkButton.tsx (hunk 6 above)
Comment (GurdeepSSingh) [reply src/components/NetworkButton/NetworkButton.tsx]: Done

File: src/components/Menu/index.tsx (hunk 5 above)
Comment (GurdeepSSingh) [reply src/components/Menu/index.tsx]: Done

File: src/views/Bills/components/Menu/index.tsx [hunk 14]
```diff
@@ -49,10 +49,7 @@ const BillMenu: React.FC<ListViewProps> = ({ onHandleQueryChange, onSetSortOptio
         </Select>
       </Flex>
       <Link href="?modal=bills" shallow>
-        <Button
-          csx={{ height: '36px', width: '192px', border: '2px solid', justifyContent: 'center' }}
-          variant="secondary"
-        >
+        <Button sx={{ LearnMoreButton }} variant="secondary">
```
Comment (kil-san) [inline src/views/Bills/components/Menu/index.tsx]: This is not how `sx` works, should be `sx={LearnMoreButton}`

File: src/components/LiquidityWidget/ConnectButton.tsx (hunk 1 above)
Comment (kil-san) [reply src/components/LiquidityWidget/ConnectButton.tsx:15]: This does not look done

**Review by kil-san — CHANGES_REQUESTED** [review]

File: src/components/Menu/components/MobileMenu.tsx (hunk 13 above)
Comment (kil-san) [reply src/components/Menu/components/MobileMenu.tsx]: This is not done

**Review by kil-san — CHANGES_REQUESTED** [review]

File: src/components/Menu/index.tsx [hunk 15]
```diff
@@ -103,29 +100,15 @@ const Menu: React.FC<{ chain?: number }> = () => {
...
-          height: '60px',
-          transform: `translateY(${showMenu ? 0 : '-100%'})`,
-          transition: 'transform linear 0.1s',
-          zIndex: 9,
-        }}
-      >
-        <Flex
-          sx={{ alignItems: 'center', height: '100%', justifyContent: 'space-between', px: ['25px', '25px', '20px'] }}
-        >
+      <Box sx={buttonStyle}>
```
Comment (kil-san) [inline src/components/Menu/index.tsx]: `buttonStyle` name does not match the component. It is not a button

File: src/components/Menu/index.tsx [hunk 16]
```diff
@@ -143,21 +126,7 @@ const Menu: React.FC<{ chain?: number }> = () => {
...
-            width: '100%',
-            height: '100%',
-            top: 0,
-            left: 0,
-            zIndex: 5,
-            background: 'rgb(56, 56, 56, 0.6)',
-          }}
-        />
-      )}
+      {!collapse && <Box onClick={() => setCollapse(true)} sx={styles.last} />}
```
Comment (kil-san) [inline src/components/Menu/index.tsx]: this is an overlay, not sure why the style is called last

**Review by kil-san — CHANGES_REQUESTED** [review]

File: src/components/LiquidityWidget/ConnectButton.tsx (hunk 1 above)
Comment (GurdeepSSingh) [reply src/components/LiquidityWidget/ConnectButton.tsx:15]: I have added it now

File: src/views/Bills/components/Menu/index.tsx (hunk 14 above)
Comment (GurdeepSSingh) [reply src/views/Bills/components/Menu/index.tsx]: I have added it like mentioned

File: src/components/Menu/index.tsx (hunk 16 above)
Comment (GurdeepSSingh) [reply src/components/Menu/index.tsx]: Changed name  to overlay

File: src/components/Menu/index.tsx (hunk 15 above)
Comment (GurdeepSSingh) [reply src/components/Menu/index.tsx]: Changed name to menuStyle

**Pull request merged**